src/scss/main.scss

```diff
@@ -8,3 +8,7 @@
 // 2. Base stuff
 @import 'base/reset';
 @import 'base/typography';
+@import 'base/grid';
+
+// 3. Styling components
+@import 'components/textblockcomp'
```

Comment:
Use - to separate compound words in SCSS files

'text-block-comp

Reply:
Done!!!